src/main/resources/assets/jei/lang/de_DE.lang

```diff
@@ -1,64 +1,88 @@
 # Tooltips
-jei.tooltip.config=Konfiguration
+jei.tooltip.config=JEI-Konfiguration
+jei.tooltip.show.recipes=Zeige Rezepte
 jei.tooltip.show.all.recipes=Zeige alle Rezepte
 jei.tooltip.delete.item=Klicke zum Löschen
 jei.tooltip.liquid.amount.with.capacity=%,d / %,d mB
 jei.tooltip.liquid.amount=%,d mB
+jei.tooltip.transfer=Bewege Gegenstände
+jei.tooltip.recipe.ore.dict=Akzeptiert: %s
+jei.tooltip.item.colors=Farben: %s
+jei.tooltip.shapeless.recipe=Formloses Rezept
+jei.tooltip.cheat.mode=Cheat-Modus
 
 # Error Tooltips
-jei.tooltip.error.recipe.transfer.missing=Fehlende Items
-jei.tooltip.error.recipe.transfer.inventory.full=Inventar ist voll
+jei.tooltip.error.recipe.transfer.missing=Fehlende Gegenstände
```

Comment:
The shortage of gui space can be fixed, as the gui is part of the program and programs are programmed by programmers. ;-) The day when the monitor's were tiny and the memory could be counted with one hand lies decades behind us. It's just something we have to ask for, or in case of mods, program by ourselves.
There's a lot of what is "commonly accepted" in German nowadays, but only by people who think they can't change anything, as you, or people who don't think at all. But the problem of degenerating language is not unique to German, it's everywhere. I just don't want to support it, as I don't find one word or language "better" than another.
So we're at the beginning: It's not a good reason to not translate it, as it's just your opinion.